data/STRchive-loci.json

```diff
@@ -4670,4 +4670,68 @@
   "disease_tags": [],
   "references": ["pmid:38714868", "pmid:38714869", "omim:609893"],
   "additional_literature": []
+},
+{
+  "chrom": "chr18",
+  "start_hg38": 666891,
+  "stop_hg38": 667632,
+  "start_hg19": 666891,
+  "stop_hg19": 667632,
+  "start_t2t": 821216,
+  "stop_t2t": 821905,
+  "id": "CPUM_TYMS",
+  "disease_id": "CPUM",
+  "gene_strand": "-",
+  "reference_motif_reference_orientation": ["GATGGT"],
+  "pathogenic_motif_reference_orientation": ["GATGGT"],
+  "pathogenic_motif_gene_orientation": ["ACCATC"],
+  "benign_motif_reference_orientation": [],
+  "benign_motif_gene_orientation": [],
+  "unknown_motif_reference_orientation": [],
+  "unknown_motif_gene_orientation": [],
+  "disease": "Congenital Progressive Universal Melanosis",
+  "gene": "TYMS",
+  "flank_motif": null,
+  "locus_structure": "(GATGGT)*",
+  "inheritance": ["AR"],
+  "type": "Non-coding",
+  "location_in_gene": "Intron 3",
+  "benign_min": 42,
+  "benign_max": 172,
+  "intermediate_min": null,
+  "intermediate_max": null,
+  "pathogenic_min": 210,
+  "pathogenic_max": 259,
+  "ref_copies": null,
+  "motif_len": 6,
+  "age_onset": "0 (birth) [@doi:10.3892/br.2025.2016]",
+  "age_onset_min": 0.0,
+  "age_onset_max": 0.0,
+  "typ_age_onset_min": null,
+  "typ_age_onset_max": null,
+  "novel": "ref",
+  "mechanism": "LoF",
+  "mechanism_detail": "Proposed mechanism involves repeat expansions in non-coding regions of the gene, reducing expression in melanocytes or keratinocytes, leading to a disruption in nucleotide balance in DNA repair and hyperpigmentation [@doi:10.3892/br.2025.2016].",
+  "source": [],
+  "details": "Missense mutations disrupt nucleotide metabolsim, resulting in loss-of-function and genome instability [@doi:10.3892/br.2025.2016].",
```

Comment:
This information in details seems to belong in the mechanism_detail section. Here you could include some information about the tandem repeat itself. For example the information used to estimate the benign and pathogenic ranges and that the parents were heterozygous.